### PR TITLE
[7.67.x-blue][JBPM-10130] - Fix wrong alter statements for some databases

### PR DESCRIPTION
**Pull request by afalhambra**
**[JBPM-10130](https://issues.redhat.com/browse/JBPM-10130)**:

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by afalhambra** [comment]
@gmunozfe this has been tested and works fine in our CI for 7.67.x-blue

**Review by VaniHaripriya — APPROVED** [review]
Looks good to me. Thank you Antonio.

**Review by gmunozfe — APPROVED** [review]
Looks good to me, great work @afalhambra !

**Comment by afalhambra** [comment]
@mareknovotny could you please merge? this is needed to fix issue for reopened https://issues.redhat.com/browse/RHPAM-4253

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/hsqldb/rhpam-7.13-to-7.13.1.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ALTER TABLE TimerMappingInfo ADD COLUMN processInstanceId bigint;
 ALTER TABLE TimerMappingInfo ALTER COLUMN timerId bigint;
 DROP INDEX EventTypes.IDX_EventTypes_element;
-CREATE INDEX IDX_EventTypes_IdElement ON EventTypes(InstanceId,element);
\ No newline at end of file
+CREATE INDEX IDX_EventTypes_IdElement ON EventTypes(InstanceId,element);
```
Comment (mareknovotny) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/hsqldb/rhpam-7.13-to-7.13.1.sql:4]: what is this line changing? a whitespace?

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/h2/rhpam-7.13-to-7.13.1.sql [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ALTER TABLE TimerMappingInfo ADD COLUMN processInstanceId bigint;
 ALTER TABLE TimerMappingInfo ALTER COLUMN timerId bigint;
 DROP INDEX IDX_EventTypes_element;
-CREATE INDEX IDX_EventTypes_IdElement ON EventTypes(InstanceId,element);
\ No newline at end of file
+CREATE INDEX IDX_EventTypes_IdElement ON EventTypes(InstanceId,element);
```
Comment (mareknovotny) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/h2/rhpam-7.13-to-7.13.1.sql:4]: what is this line changing? a whitespace?

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/derby/rhpam-7.13-to-7.13.1.sql [hunk 3]
```diff
@@ -1,4 +1,4 @@
 ALTER TABLE TimerMappingInfo ADD COLUMN processInstanceId bigint;
 ALTER TABLE TimerMappingInfo ALTER COLUMN timerId bigint;
 DROP INDEX EventTypes.IDX_EventTypes_element;
-CREATE INDEX IDX_EventTypes_IdElement ON EventTypes(InstanceId,element);
\ No newline at end of file
+CREATE INDEX IDX_EventTypes_IdElement ON EventTypes(InstanceId,element);
```
Comment (mareknovotny) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/derby/rhpam-7.13-to-7.13.1.sql:4]: what is this line changing? a whitespace?

**Comment by mareknovotny** [comment]
we need to wait with merging as the build was already handed over so there is no agreement on having this now included

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/hsqldb/rhpam-7.13-to-7.13.1.sql (hunk 1 above)
Comment (afalhambra) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/hsqldb/rhpam-7.13-to-7.13.1.sql:4]: It's just a new line removed. Just to be in sync completely with the same file we have in 7.67.x branch, though no major impact out of this change.

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/h2/rhpam-7.13-to-7.13.1.sql (hunk 2 above)
Comment (afalhambra) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/h2/rhpam-7.13-to-7.13.1.sql:4]: same as previous

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/derby/rhpam-7.13-to-7.13.1.sql (hunk 3 above)
Comment (afalhambra) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/derby/rhpam-7.13-to-7.13.1.sql:4]: same as previous

**Pull request merged**